pyccel/codegen/printing/ccode.py

```diff
@@ -1089,6 +1084,8 @@ def _print_Import(self, expr):
         if source == 'stc/common':
             code = ''
             for t in expr.target:
+                self.add_import(Import(stc_extension_mapping[source],(),
+                       ignore_at_print=True))
```

Comment:
```suggestion
                self.add_import(Import(stc_extension_mapping[source], ignore_at_print=True))
```

Reply:
The `target` argument is optional. Better to let `Import` decide on the default

Reply:
Got it! I will commit your suggestion as soon as the test finishes.

Reply:
Oops. Sorry it looks like I misremembered/misread that :eyes: